frontend/src/components/tree/Imageslider.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react'
+import { ArrowRight, ArrowLeft } from 'lucide-react'
+
+interface ImageSliderProps {
+  images: string[]
+}
+
+const Imageslider: React.FC<ImageSliderProps> = ({ images }) => {
+  const [currentIndex, setCurrentIndex] = useState(0)
+  const [visibleImages, setVisibleImages] = useState(1)
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth >= 1024) {
+        setVisibleImages(4) // For large screens (desktop)
+      } else if (window.innerWidth >= 768) {
+        setVisibleImages(3) // For medium screens (tablet)
+      } else {
+        setVisibleImages(1) // For small screens (phone)
+      }
+    }
+
+    handleResize()
+
+    window.addEventListener('resize', handleResize)
+
+    return () => window.removeEventListener('resize', handleResize)
+  }, [])
+
+  const goToPrevious = () => {
+    if (currentIndex > 0) {
+      setCurrentIndex((prevIndex) => prevIndex - 1)
+    }
+  }
+
+  const goToNext = () => {
+    if (currentIndex < images.length - visibleImages) {
+      setCurrentIndex((prevIndex) => prevIndex + 1)
+    }
+  }
+
+  const displayedImages = images.slice(
+    currentIndex,
+    currentIndex + visibleImages
+  )
+
+  const totalImages = images.length
+  const startIndex = currentIndex + 1
+  const endIndex =
+    currentIndex + visibleImages <= totalImages
+      ? currentIndex + visibleImages
+      : totalImages
+
+  const rangeText =
+    visibleImages === 1
+      ? `${startIndex} von ${totalImages} Bildern`
+      : `${startIndex}-${endIndex} von ${totalImages} Bildern`
+
+  const shouldShowNavigation = images.length > visibleImages
+
+  return (
+    <div className="relative w-full">
+      <h2 className="font-bold text-xl mb-4">Hinterlegte Bilder:</h2>
+
+      <div className="flex space-x-4 mb-4 w-full">
+        {displayedImages.map((image, index) => (
+          <img
+            key={index}
+            src={image}
+            alt={`Slide ${currentIndex + index + 1}`}
+            className="w-full sm:w-1/3 md:w-1/4 h-48 sm:h-64 md:h-72 object-cover rounded-lg shadow-lg"

```

Comment:
Please do not use a fixed height. Instead you can use aspect-ratio. That will insure, that the image has always the same ratio of width to height 👍